_sitemap/sitemap.xml

```diff
@@ -1,314 +1,314 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+<urlset xmlns="https://www.sitemaps.org/schemas/sitemap/0.9">
```

Comment:
Namespaces are not URLs; they should not change protocols.

Reply:
I actually doubt this will fix anything.  Lets just close...